opennms-doc/guide-admin/src/asciidoc/text/operation/jmx-config-generator/webui.adoc

```diff
@@ -0,0 +1,80 @@
+
+// Allow GitHub image rendering
+:imagesdir: ../../../images
+
+=== Web based JMX Configuration Generator
+
+Generating complex _JMX_ data collection configurations can be generated from the a web based tool.
+It collects all available _MBean Attributes_ or _Composite Data Attributes_ from a _JMX_ enabled Java application.
+
+The workflow of the tool is:
+
+. Connect with _JMX_ or _JMXMP_ against a _MBean Server_ provided from a _Java_ application
+. Retrieve all _MBean_ and _Composite Data_ from the application
+. Select specific _MBeans_ and _Composite Data_ objects which should be collected by _OpenNMS_
+. Generate _JMX Collectd_ configuration file and _RRD graph_ definitions for _OpenNMS_ as downloadable archive
+
+The following connection settings are supported:
+
+* Ability to connect to _MBean Server_ with _RMI_ based _JMX_
+* Authentication credentials for _JMX_ connection
+* Optional: JMXMP connection
+
+The web based configuration tool can be used in the _OpenNMS Web Application_ in administration section _Admin -> JMX Configuration Generator_.
+
+==== Configure JMX Connection
+
+At the beginning the connection to an _MBean Server_ of an _Java_ application has to be configured.
+
+.JMX connection configuration window
+image::operation/jmx-config-generator/01-webui-connection.png[]
+
+* _Service name_: The name of the service to bind the _JMX_ data collection for _Collectd_
+* _Host_: IP address or _FQDN_ connecting to the _MBean Server_ and load _MBeans_ and _Composite Data_ into the generation tool
+* _Port_: Port to connect to the _MBean Server_
+* _authenticate_: Enable / Disable authentication for _JMX_ connection with username and password
+* _Skip JVM MBeans_: Enable / Disable retrieving generic _Java JVM MBeans_
+* _Run writable MBeans_: Enable / Disable retrieving writable _MBean Attributes_
```

Comment:
Maybe add a hint, that even if they are readable, they are skipped if they are also writable. This option is to force retrieving these MBeans as well.